songbird/integration/hikari.py

```diff
@@ -46,44 +64,44 @@ async def initialize(
             user_id=user_id
         )
 
-        self = Voicebox(driver)
+        class_ = cls(driver)
```

Comment:
imo `self` is a much clearer variable name here

Reply:
It isnt self, its class object, although maybe you're right